### PR TITLE
predict! and fixed issue with sparse arrays of importance

### LINKED ISSUE #87: new method: `predict`

### PR DESCRIPTION
**Pull request by nbokulich**
fixes #87

### CODE REVIEW

File: q2_sample_classifier/utilities.py [hunk 1]
```diff
@@ -126,6 +127,8 @@ def _extract_important_features(index, top):
     # is top a 1-d or multi-d array?
     # coef_ is a multidimensional array of shape = [n_class-1, n_features]
     if any(isinstance(i, list) for i in top) or top.ndim > 1:
+        if issparse(top):
+            top = top.todense()
```
Comment (ebolyen) [inline q2_sample_classifier/utilities.py]: That is a straight forward fix!

Comment (nbokulich) [reply q2_sample_classifier/utilities.py]: indeed! I was spending too much time trying to figure out why (a) it only affected some methods and (b) interacting with the sparse matrix. In the end it was so much easier to just convert to dense (feature importance scores should not be as massive as the full dataset so I'm hoping this doesn't cause memory issues)

**Comment by ebolyen** [comment]
@nbokulich, looks like flake8 is mad

**Comment by nbokulich** [comment]
oops sorry @ebolyen in all my excitement I forgot to check. Fixed!

**Pull request merged**